### ISSUE TITLE
character conversion failed

### ISSUE THREAD

**Issue by substack** [systemed/tilemaker#64]
I'm running debian so I couldn't follow the ubuntu instructions, but I installed the latest version of boost (1.62.0) and symlinked it into `./include`. Everything compiles successfully, but I get this error:

```
$ ./tilemaker 
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::logic_error> >'
  what():  character conversion failed
Aborted
```

**Comment by substack** [comment]
I get the same error if I provide a file and an output directory:

```
$ ./tilemaker vancouver_canada.osm.pbf --output=/tmp/vancouver
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<std::logic_error> >'
  what():  character conversion failed
Aborted
```

**Comment by systemed** [comment]
Looks like this might be a boost issue: https://svn.boost.org/trac/boost/ticket/11905 . I'll see if we can work around it.

**Comment by substack** [comment]
I figured out the problem in case somebody else runs into this: I had a mismatch between the system-wide boost libs that tilemaker links against and the boost source files. I didn't see that I had to run the boost `bootstrap.sh` command followed by `./b2 --prefix=$PREFIX` and then I had to `cp -P libs/* ~/prefix/lib && cp -r boost ~/prefix/include`. Then I edited the tilemaker Makefile to add `-L$(PREFIX)/lib` to LIB and `-I$(PREFIX)/include` to INC.
